### PR TITLE
TDB-1892 adds getClusterTierName() method to Cluster class

### PR DESCRIPTION
**Pull request by gibsong**
I added a helper function to the Cluster class.  The function is getClusterTierName(String clientId, String cacheAlias), which is used to extract a the clusterTierName from the topology by using the clientId and the cacheAlias.  This method is needed in the StatisticsService and the TmsClusterMessageListener.

### CODE REVIEW

File: management/cluster-topology/src/main/java/org/terracotta/management/model/cluster/Cluster.java [hunk 1]
```diff
@@ -186,6 +187,33 @@ public String getPath(Context context) {
     return stripeStream().flatMap(Stripe::serverStream);
   }
 
+  public String getClusterTierName(String clientId, String cacheAlias) {
```
Comment (mathieucarbou) [inline management/cluster-topology/src/main/java/org/terracotta/management/model/cluster/Cluster.java]: @gibsong : the concept of cluster tier is a special concept of Ehcache. It is not related at all to this model, which only support a raw topology view of a cluster. This model doesn't have any interpretation of the data it holds.

Such computation must be done in the project where you are using the topology, not within the topology itself.

Comment (gibsong) [inline management/cluster-topology/src/main/java/org/terracotta/management/model/cluster/Cluster.java]: Shoot ok I will put this method in another class in the terracotta-enterprise project.  I thought this was the best place to add it but I guess I was wrong.

**Comment by mathieucarbou** [comment]
I will close this PR

**Comment by gibsong** [comment]
ok thanks for closing

**Pull request closed without merging**